.github/workflows/ci.yaml

```diff
@@ -8,40 +8,39 @@ on:
     branches:
       - master
 
+env:
+  POETRY_VIRTUALENVS_CREATE: "false"
+
 jobs:
   default:
-    name: "${{ matrix.step.env }} '${{ matrix.step.command }}' @ py${{ matrix.step.python }})"
+    name: "${{ matrix.step.env }} '${{ matrix.step.command }}' (@py${{ matrix.step.python }})"
     runs-on: ubuntu-latest
     strategy:
       matrix:
         step: [
           # original from drone CI startlark file
           # {env: "pytest", python: "3.6", extras: "main,dev", command: "pytest tests/"},  # disable or drop support?
-          {env: "pytest", python: "3.7", extras: "main,dev", command: "pytest tests/"},
-          {env: "pytest", python: "3.8", extras: "main,dev", command: "pytest tests/"},
-          {env: "pytest", python: "3.9", extras: "main,dev", command: "pytest tests/"},
-          {env: "main", python: "3.8", extras: "main,dev", command: "flakeheaven lint"},
-          {env: "typing", python: "3.8", extras: "main,dev", command: "mypy --ignore-missing-imports --allow-redefinition flakeheaven/ > /dev/null || yes | mypy --install-types && mypy --ignore-missing-imports --allow-redefinition flakeheaven/"},
+          {env: "pytest", python: "3.7", extras: "", command: "pytest tests/"},
+          {env: "pytest", python: "3.8", extras: "", command: "pytest tests/"},
+          {env: "pytest", python: "3.9", extras: "", command: "pytest tests/"},
+          {env: "main", python: "3.8", extras: "", command: "flakeheaven lint"},
+          {env: "typing", python: "3.8", extras: "", command: "mypy --ignore-missing-imports --allow-redefinition flakeheaven/ > /dev/null || yes | mypy --install-types && mypy --ignore-missing-imports --allow-redefinition flakeheaven/"},
           # additional
-          {env: "docs", python: "3.8", extras: "docs", command: "sphinx-build -W docs docs/build"},
-          {env: "isort", python: "3.8", extras: "main,dev", command: "isort --check flakeheaven"},
+          {env: "docs", python: "3.8", extras: "-E docs", command: "sphinx-build -W docs docs/build"},
+          {env: "isort", python: "3.8", extras: "", command: "isort --check flakeheaven"},
         ]
     container:
-      image: python:${{ matrix.step.python }}-alpine
+      image: python:${{ matrix.step.python }}-bullseye
```

Comment:
installing poetry was giving some gcc error in alpine, switched to debian

Reply:
Where I've used Alpine in other work, I end up adding the following to the Docker image to get gcc to work (fix ffi and other errors). However, if it works fine in Debian, then there's probably little benefit to using Alpine. 
```
    apk add --no-cache --upgrade --virtual .build-deps \
        build-base \
        cargo \
        libffi-dev \
        libxml2-dev \
        libxslt-dev \
        openssl-dev \
        python3-dev \
        py3-wheel && \
```

Reply:
Only build time and size...

Reply:
yeah my thinking was that the less steps we had here the better, and switching to debian meant no needed to install any extra dependencies 🙂 lmk if y'all have any concerns and I could switch back

Reply:
agreed, less is more :)